rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1057,12 +1053,40 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	}
 
 	ui->searchBox->setPlaceholderText(tr("Search libraries", "Library search box"));
+	ui->searchBox->setEnabled(true);
+	ui->lleList->setEnabled(true);
 
 	auto on_lib_button_clicked = [this](int id)
 	{
-		const bool enableLibs = id != static_cast<int>(lib_loading_type::liblv2only);
-		ui->searchBox->setEnabled(enableLibs);
-		ui->lleList->setEnabled(enableLibs);
+		std::vector<QListWidgetItem*> items;
+
+		// duplicate current items, we need clones to preserve checkstates
+		for (int i = 0; i < ui->lleList->count(); i++)
+		{
+			items.push_back(ui->lleList->item(i)->clone());
+		}
+
+		for (uint i = 0; i < items.size(); i++)
+		{
+			const auto text = sstr(items[i]->text());
+
+			// Select default LLE sprx list for manual lib loading, otherwise all libs are unchecked
+			items[i]->setCheckState(id == static_cast<int>(lib_loading_type::manual) && g_prx_list.count(text) && !g_prx_list.at(text)? Qt::Checked : Qt::Unchecked);
```

Comment:
This is more readable imo:
```suggestion
			const bool selected = id == static_cast<int>(lib_loading_type::manual) && g_prx_list.count(text) && !g_prx_list.at(text);
			items[i]->setCheckState(selected ? Qt::Checked : Qt::Unchecked);
```